### DISCUSSION TITLE
I've a very low IQ and I am placed in special ED classes, but I still can never even remember or understand anything. What should I do?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#110879] [Programming Help]
### Body

Provide a simple answer

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by P09s** [comment]
It's important to recognize that intelligence is not solely determined by IQ or academic performance. Each individual has unique strengths and abilities, and it's essential to focus on discovering and nurturing those strengths.

Remember that everyone learns at their own pace, and there is no one-size-fits-all approach to learning. Keep exploring different strategies and seeking support, and don't hesitate to advocate for yourself in your educational journey. You are capable of achieving success in your own unique way.

By the way I am also an engineering student and studied ED too !!

**Reply by ghost** [reply]
I am not an engineering student in a way like I went to school to learn coding I learned on the internet by starting with languages like VB.net, C#, CSS, HTML, JavaScript, C++, Python, PHP, MySQL and finally Assembly specifically for the X86 architecture.

**Reply by P09s** [reply]
Well that's great ✨

**Comment by davea42** [comment]
Congratulations @PyDOS8 on your accomplishments.  We'll all be watching your future queries with the hope we can provide some useful advice.
